src/components/application_manager/src/commands/mobile/send_location_request.cc

```diff
@@ -220,6 +220,18 @@ bool SendLocationRequest::IsWhiteSpaceExist() {
   return false;
 }
 
+bool IsSendLocationEnabled(const HMICapabilities& hmi_capabilities) {
```

Comment:
@LitvinenkoIra 
Better wrap it with anonymous namespace to not mess common 'commands' namespace

Reply:
@dev-gh Done 3c764f165ae2d3ae1288300f21d7dd3ecd98a4f8